image_feed/left: simulator/camera1/image_raw
image_feed/right: simulator/camera2/image_raw
camera_sensor_height: 0.04
camera_focal_length: 0.02
#Gate-related parameters
gate_hue_range1_start: 125
gate_hue_range1_end: 179
gate_hue_range2_start: 0
gate_hue_range2_end: 20
gate_value_start: 55
gate_value_end: 200
